server/src/test/java/io/crate/expression/scalar/object/ObjectMergeFunctionTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.object;
+
+import java.util.Map;
+
+import org.junit.Test;
+
+import io.crate.expression.scalar.ScalarTestCase;
+
+public class ObjectMergeFunctionTest extends ScalarTestCase {
+
+    @Test
+    public void test_null_input() {
+        assertEvaluate("object_merge(null, null)", null);
```

Comment:
not sure how to assert the `null` cases best. 
This currently fails:
`java.lang.IllegalArgumentException: No assertions group should be null`

Reply:
Please use `assertEvaluateNull`, this is because of the recent assertj migration: https://github.com/crate/crate/commit/a9e3d857c43f50c76c782fd8ad5e5bf30257db53